### PR TITLE
FWK-777 - fix list index out of range

### PR DESCRIPTION
**Pull request by pploegaert**

### CODE REVIEW

**Comment by JeffreyDevloo** [comment]
Edit: No longer approved. After re-evaluating the code, I made a mistake on what the issue could be.
Need to test this fix further before approving.

**Comment by khenderick** [comment]
Just wondering; did you guys test the rest of the changed code? Is the `dtl_checkup` still doing what it's supposed to do when this list would end up empty?

File: ovs/lib/vdisk.py [hunk 1]
```diff
@@ -908,7 +908,9 @@ def dtl_checkup(vpool_guid=None, vdisk_guid=None, storagerouters_to_exclude=None
                 current_sr = None
                 available_primary_srs = available_primary_srs.difference(available_secondary_srs)
                 if current_dtl_config is not None:
-                    current_sr = [sd for sd in vpool.storagedrivers if sd.storage_ip == current_dtl_config.host][0].storagerouter
+                    current_srs = [sd for sd in vpool.storagedrivers if sd.storage_ip == current_dtl_config.host]
```
Comment (khenderick) [inline ovs/lib/vdisk.py:911]: `current_srs` should be `current_sds`, as we're working with StorageDrivers here, not StorageRouters.

File: ovs/lib/vdisk.py [hunk 2]
```diff
@@ -908,7 +908,9 @@ def dtl_checkup(vpool_guid=None, vdisk_guid=None, storagerouters_to_exclude=None
...
+                    if current_srs:
```
Comment (JeffreyDevloo) [inline ovs/lib/vdisk.py:912]: I'd opt for len(current_srs)>=1 for more clarification what its about

**Comment by khenderick** [comment]
Closed since this PR only fixes the error itself, and not the reason why it was caused. Will be resolved by another PR implementing the discussed requirements in #777

**Pull request closed without merging**